app/src/main/java/com/google/samples/apps/nowinandroid/navigation/NiaNavHost.kt

```diff
@@ -39,33 +39,32 @@ import com.google.samples.apps.nowinandroid.ui.NiaAppState
 @Composable
 fun NiaNavHost(
     appState: NiaAppState,
+    snackbarHostState: SnackbarHostState,
     modifier: Modifier = Modifier,
     startDestination: String = forYouNavigationRoute,
 ) {
     val navController = appState.navController
+    val interestsScrollState = rememberLazyGridState()
```

Comment:
The grid state needs to be hoisted outside the navigation destination, otherwise, the left pane will not retain it's scroll state when navigating from topic to topic. See attached video.

[Screen_recording_20230531_150631.webm](https://github.com/android/nowinandroid/assets/6339497/7831b3c3-5f94-431d-9188-ccbcc12e149d)

Reply:
Hey Manu, as we agreed in our conversation, this is the first place we can hoist the scroll state.